packages/flash-calendar/src/components/CalendarItemDay.tsx

```diff
@@ -126,7 +126,8 @@ const buildBaseStyles = (theme: BaseTheme): CalendarItemDayTheme => {
   };
 };
 
-export interface CalendarItemDayProps {
+export interface CalendarItemDayProps
+  extends Omit<TextProps, "children" | "onPress"> {
```

Comment:
since `CalendarItemDay` and `WeekName` components are not direct wrappers of Text, I believe we need to do something like:
```suggestion
  textProps: Omit<TextProps, "children" | "onPress">;
```

This preserves optionality in the case we need to add something like `pressableProps`.

Reply:
@MarceloPrado Great suggestion, I pushed the adjustments.  Please take another look